src/lib/state.ts

```diff
@@ -380,13 +384,14 @@ export const dumpNode = (node: ZWaveNode, schemaVersion: number): NodeState => {
     individualEndpointCount: node.individualEndpointCount,
     aggregatedEndpointCount: node.aggregatedEndpointCount,
     interviewAttempts: node.interviewAttempts,
-    interviewStage: node.interviewStage,
     endpoints: Array.from(node.getAllEndpoints(), (endpoint) =>
       dumpEndpoint(endpoint, schemaVersion)
     ),
     values: getNodeValues(node, schemaVersion),
   };
 
+  if (schemaVersion <= 3) base.interviewStage = node.interviewStage;
```

Comment:
you can put this in the existing if-statement below?

Reply:
The statement below is `< 3`, this is `<=3`

Reply:
oh :)